src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-permissions.mdx

```diff
@@ -291,6 +291,7 @@ These permissions pertain to [synthetics monitoring](/docs/synthetics/synthetic-
 * <DoNotTranslate>**Configure private locations**</DoNotTranslate>: relates to [private locations](/docs/synthetics/synthetic-monitoring/private-locations/private-locations-overview-monitor-internal-sites-add-new-locations). 
 * <DoNotTranslate>**Monitor downtimes**</DoNotTranslate>: relates to [monitor downtimes](/docs/synthetics/synthetic-monitoring/using-monitors/monitor-downtimes-disable-monitoring-during-scheduled-maintenance-times).
 * <DoNotTranslate>**Monitor scripts**</DoNotTranslate>: relates to [scripted monitors](/docs/synthetics/synthetic-monitoring/getting-started/types-synthetic-monitors) (scripted browser monitors and scripted API test monitors). 
+* <DoNotTranslate>**Monitors**</DoNotTranslate>: relates to ability to configure [synthetic monitors](/docs/synthetics/synthetic-monitoring/getting-started/types-synthetic-monitors) (for example, name, period, locations). The bulk runtime upgrade permission allows you to make runtime changes using the [runtime upgrades UI](/docs/synthetics/synthetic-monitoring/using-monitors/runtime-upgrade-ui/).
```

Comment:
there's an and or an or missing between period, and locations I think

Reply:
Good catch! Thank you!